editors/code/src/ctx.ts

```diff
@@ -20,6 +20,7 @@ export class Ctx {
         const res = new Ctx(config, extCtx, client, serverPath);
         res.pushCleanup(client.start());
         await client.onReady();
+        client.onRequest('workspace/configuration', _ => [configToServerOptions(config)]);
```

Comment:
Similarly, is there some kind of build-in handler here, which just fetches the relevant section from the config?